backend/routes/users.js

```diff
@@ -21,6 +21,72 @@ router.get("/all", function(req, res) {
     });
 });
 
+// Get a user's programs
+// Returns an array of programs: [{programId, programName, role}]
+router.get("/:userId/programs", function(req, res) {
+  db["Authentication"].users
```

Comment:
I don't think this returns the users role.

Reply:
I updated this yesterday! You might've been looking at an old commit

Reply:
This is what i am getting for a return value

```
    {
        "programId": "5f54b07f9971a3dd4f742328",
        "programName": "Full Proposal Applications"
    },
    {
        "programId": "5fab02657cf3c7788fc00d1f",
        "programName": "Demo2"
    },
```

Reply:
That's weird, I get the role in my latest commit. Could you try pulling this branch again?

Reply:
I think I figured out why. It is returning the wrong "role" field. i.e 
```
{"_id":{"$oid":"5e7ca0c3412aeae89f5eb81f"},"userId":"vBUgTex5MeNd57fdB8u4wv7kXZ52","__v":0,"email":"gmaxin@uwblueprint.org","name":"Gregster","programs":[{"role":"reviewer","_id":{"$oid":"5fc1816e64f8300924df0ca9"},"id":"5f54af1b9971a3dd4f73e451"},{"role":"reviewer","_id":
{...}
{"$oid":"5fc1816e64f8300924df0cb5"},"id":"5fc16f497cf3c7788f310a26"}],"role":"Milk","admin":false,"currentProgram":"5f54af1b9971a3dd4f73e451"}
```
returns 
```
{
        "role": "Milk",
        "programId": "5f54af1b9971a3dd4f73e451",
        "programName": "Emergency Community Support Fund (ECSF) Round 2"
    },
    {
        "role": "Milk",
        "programId": "5f54b04d9971a3dd4f741a9e",
        "programName": "Emergency Community Support Fund (ECSF) Round 1"
    },
...
```

Whereas use `wTseaXghLmQpKWQ3H9dK5laC03N2` doesn't have a role field in its user so it doesn't show it.

Reply:
Ahh nice catch, just pushed a fix